testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -315,6 +316,9 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
             .maybeExpectLegacyDeprecationWarningIf(FIRST_VERSION_UNDER_DEVELOCITY_BRAND <= versionNumber,
                 "WARNING: The following functionality has been deprecated and will be removed in the next major release of the Develocity Gradle plugin. " +
                     "For assistance with migration, see https://gradle.com/help/gradle-plugin-develocity-migration.")
+            .maybeExpectLegacyDeprecationWarningIf(FIRST_VERSION_UNDER_DEVELOCITY_BRAND <= versionNumber,
```

Comment:
I needed to add the assertion for the new style of deprecation message to fix the smoke tests.